src/sentry/static/sentry/app/views/organizationDiscover/styles.jsx

```diff
@@ -92,6 +93,45 @@ export const Sidebar = styled(props => (
   padding-top: ${HEADER_HEIGHT}px;
 `;
 
+export const QueryFieldsSidebar = styled(Flex)`
+  height: 100%;
+  flex-direction: column;
+  justify-content: flex-start;
+`;
+
+export const FieldSeparator = styled('div')`
+  margin: ${space(4)} ${space(3)} 0 ${space(3)};
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+`;
+
+export const DocsSeparator = styled('div')`
+  flex-grow: 1;
+  margin: ${space(3)} ${space(3)} 0;
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+`;
+
+export const StyledInlineSvg = styled(InlineSvg)`
+  justify-content: flex-end;
+`;
+
+export const DiscoverDocs = styled('span')`
```

Comment:
Would be great to reduce the font size of this component - right now it's a bit larger than the rest of the text in this component

Reply:
thanks!!